index.html

```diff
@@ -10,16 +10,23 @@
 </head>
 <body>
     <header>
-            <h2 class="logo"><a href="#">Logo</a></h2>
+            <h2 class="logo"><a href="#main-section">Moses</a></h2>
+            <nav class="nav-link">
+                <ul>
+                    <li><a href="#portfolio-works">Portfolio</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Contact">Contact</a></li>
+                </ul>
+            </nav>
             <div class="menu">
                 <ul><a href="#"><img src="image-source/Icon - Menu.png" alt="Humberger_menu"></a></ul>
             </div>
     </header>
     <main>
-        <section class="section-1">
-            <h1>I’m Moses Glad to see you!</h1>
+        <section id="main-section" class="main-section">
+            <h1>I’m Moses <br> Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <h3>LET'S CONNECT</h3>
+            <h3 class="let">LET'S CONNECT</h3>
                 <ul class="nav-links">
                     <li><a href="https://twitter.com/Ezechukwuchibu2"><img src="image-source/twitter.svg" alt="twitter"></a></li>
                     <li><a href="https://www.linkedin.com/in/ezechukwu-chibuike/"><img src="image-source/Linkedin icon.svg" alt="Linkedin"></a></li>
```

Comment:
- [ ] [OPTIONAL] You have done a great job implementing the padding on `about-me` section. 👏 . But to stick to the Figma design, I suggest that you reduce the `padding-left`. This is because it is making the `LET'S CONNECT text` squeeze as you reduce the screen size 😞 . 

- Your design 😎 

![Capture](https://user-images.githubusercontent.com/99394374/213838054-517c70db-5729-485c-b132-673f1cea0943.PNG)

Reply:
Thanks for the suggestion @AlphaNtihinduka, will do my best work on it.